.github/actions/tfplus-python-test/action.yml

```diff
@@ -1,11 +1,11 @@
 ---
-name: tfplus-python-test
+name: tfplus-test
 description: run pytest to execute python test cases of tfplus python
 runs:
   using: 'docker'
-  image: "easydl/tfplus:tf212_dev"
+  image: "easydl/tfplus:tf213_dev"
   args:
     - "/bin/bash"
     - "-c"
     - "cd tfplus \
-&& sh dev/scripts/build_and_test.sh"
+&& sh build.sh"
```

Comment:
this seems will take a long time.
can we just trigger it, once there is  code change under tfplus project?